gordo_components/server/views/common.py

```diff
@@ -4,9 +4,9 @@
 import functools
 import timeit
 import typing
-import traceback
+from typing import List, Union, Optional
```

Comment:
General; maybe move `common` into it's own module, ie. `gordo_components.common` since we're now spanning the use of functions like `make_base_dataframe` into the server (by the base view) and `DiffBasedAnomalyDetector.anomaly` .. and maybe in general, we don't have a `shared`/`common` submodule anyway. :man_shrugging:

Reply:
Move to `models`